test/jdk/javax/swing/JToggleButton/bug4128979.java

```diff
@@ -22,63 +22,67 @@
  */
 
 /* @test
-   @bug 4128979
-   @requires (os.family == "windows")
-   @summary Tests that background changes correctly in WinLF for JToggleButton when pressed
-   @key headful
-   @run applet/manual=yesno bug4128979.html
-*/
+ * @bug 4128979
+ * @requires (os.family == "windows")
+ * @modules java.desktop/sun.awt
+ * @library /java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame jtreg.SkippedException
+ * @summary Tests that background changes correctly in WinLF for JToggleButton when pressed
+ * @run main/manual bug4128979
+ */
```

Comment:
Move it after imports.

Reply:
Updated but placing of imports is still confusing.